api/src/main/java/marquez/api/SearchResource.java

```diff
@@ -72,6 +83,48 @@ public Response search(
     return Response.ok(new SearchResults(searchResults)).build();
   }
 
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Produces(APPLICATION_JSON)
+  @Path("/jobs")
+  public Response searchJobs(@QueryParam("q") @NotBlank String query) throws IOException {
```

Comment:
We'll want to expand the query (`q`) to also allow for qualifiers. For example, to search for metadata within a given namespace, the user can provide the qualifier `namespace:<NAMESPACE>`:

```
q=etl namespace:my-namespace
```

Reply:
So yes, we wish to do this, it's actually more work to support this kind of thing on the UI side than anything because you need to provide typeahead assistance in expression completion to have this experience be fluid. Once we get some feedback here about what is interesting and what subset of the search DSL is interesting to users, we can add it.